wct.conf.js

```diff
@@ -3,8 +3,20 @@ var args = require('yargs').argv;
 module.exports = {
   extraScripts: args.dom === 'shadow' ? ['test/enable-shadow-dom.js'] : [],
   registerHooks: function(context) {
-    if (process.env.SAUCE_BROWSERS) {
-      context.options.plugins.sauce.browsers = JSON.parse(process.env.SAUCE_BROWSERS);
+    // do not run Saucelabs tests if PR branch containt 'quick/'
+    if (process.env.TRAVIS_BRANCH.indexOf('quick/') === -1) {
```

Comment:
I think we need `process.env.TRAVIS_EVENT_TYPE === 'push' && ...` here to avoid this being triggered for external pull requests?